File: 04 Technical/02 Server/01 Logics/01 Python/01 General/01 Prime Number.py
# Find Prime Numbers

given_number = 10  # int(input())
print(given_number)

print("-------------------")

prime_numbers_list = []
for main_counter in range(1, given_number+1):
    factor_count = 0
    for sub_counter in range(1, main_counter+1):
        if main_counter % sub_counter == 0:
            factor_count += 1
    if factor_count == 2:
        prime_numbers_list.append(main_counter)


print(prime_numbers_list)  # [2, 3, 5, 7]

print("===================")

File: 04 Technical/01 Client/01 Logics/01 Javascript/01 Prime Number.js
// Find Prime Numbers

given_number = 10;
console.log(given_number);

console.log('--------------------');

let prime_numbers_array = [];
for (i = 1; i <= given_number; i++) {
	let factors_count = 0;
	for (j = 1; j <= i; j++) {
		if (i % j == 0) {
			factors_count += 1;
		}
	}
	if (factors_count == 2) {
		prime_numbers_array.push(i);
	}
}

console.log(prime_numbers_array);

console.log('=======================');
